docs/Contributing/product-groups/README.md

```diff
@@ -0,0 +1,44 @@
+# Fleet Product Groups
+
+Fleet is organized into three main product groups:
+
+## MDM
+
+The [MDM product group](mdm/README.md) focuses on Mobile Device Management, including:
+
+- Apple MDM (macOS, iOS)
+- Windows MDM
+- Android MDM
+- End user authentication
+- Disk encryption
+- Automated device enrollment
+
+## Orchestration
+
+The [Orchestration product group](orchestration/README.md) focuses on device orchestration using osquery, including:
+
+- Live queries
+- Scheduled queries
+- Query packs
+- Host vitals
+- Teams and access control
+- File carving
+
+## Software
+
+The [Software product group](software/README.md) focuses on software management, including:
+
+- Software inventory
+- Vulnerability management
+- Software installation
+- Software updates
+- Software policies
```

Comment:
We're missing Fleet-maintained apps, and the documentation for both of these items is inaccurate (and means something _completely_ different for policies), so need to double-check this.

Which leads me to believe the MDM group responsibilities aren't quite right either.

Reply:
Added Fleet-maintained apps. Feel free to suggest changes with additions. I'm sure the others aren't complete yet, but this is a start that we can build on.